engine/flink/model-util/src/main/scala/pl/touk/nussknacker/engine/flink/util/transformer/aggregate/sampleTransformers.scala

```diff
@@ -101,7 +104,65 @@ object sampleTransformers {
                 @ParamName("windowLengthInSeconds") length: Long,
                 @OutputVariableName variableName: String)(implicit nodeId: NodeId): ContextTransformation = {
       val windowDuration = Duration(length, TimeUnit.SECONDS)
-      transformers.slidingTransformer(keyBy, aggregateBy, aggregator, windowDuration, variableName, explicitUidInStatefulOperators)
+      transformers.slidingTransformer(keyBy, aggregateBy, aggregator, windowDuration, variableName, emitWhenEventLeft = false, explicitUidInStatefulOperators)
     }
   }
+
+  /**
+   * This aggregator can be used for both predefined aggregators (see list below) and for some specialized aggregators like #AGG.map
+   * when you switch editor to "raw mode". It also has `emitWhenEventLeft` flag.
+   */
+  object SlidingAggregateTransformerV2 extends CustomStreamTransformer with ExplicitUidInOperatorsSupport {
+
+    @MethodToInvoke(returnType = classOf[AnyRef])
+    def execute(@ParamName("keyBy") keyBy: LazyParameter[CharSequence],
+                @ParamName("aggregator")
+                @DualEditor(simpleEditor = new SimpleEditor(
+                  `type` = SimpleEditorType.FIXED_VALUES_EDITOR,
+                  possibleValues = Array(
+                    new LabeledExpression(label = "First", expression = "T(pl.touk.nussknacker.engine.flink.util.transformer.aggregate.AggregateHelper).FIRST"),
```

Comment:
Would be nice to have something like `@EditorFactory(SomeFactory.class)` like it is with `TypeInfoFactory`. For now alternative option is to use `WithExplicitMethodToInvoke`? Does it work with custom nodes? Or we need to use `GenericNodeTransformation`? I'd rather like this code duplication because both options will generate a bit of boilerplate and can introduce desynchronization between definition and implementation...